Dockerfile

```diff
@@ -13,4 +13,3 @@ RUN bundle install
 COPY . /app
 
 EXPOSE 3000
-CMD ["bundle", "exec", "rails", "server", "-b", "0.0.0.0"]
```

Comment:
You override this in the docker compose file below, so there is no harm in leaving this here. It will allow you to spin up this single image without having to define it, sort of like a fallback.